pkg/testsuite/helpers.go

```diff
@@ -0,0 +1,85 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package testsuite
+
+import (
+	"context"
+	"crypto/rand"
+	"fmt"
+	"log"
+	"math/big"
+	"net/http"
+	"time"
+
+	"github.com/sethvargo/go-retry"
+)
+
+func randomString() (string, error) {
```

Comment:
There's a few existing `random*` helpers for making a string, most use bytes. I think we should use one of those instead.

Reply:
I found one in e2e-runner/main.go which is unexported. Maybe we can create a common util directory but for now I just copy paste it.